source/wp-content/themes/wporg-developer-2023/src/chapter-list/view.js

```diff
@@ -0,0 +1,44 @@
+const init = () => {
+	const container = document.querySelector( '.wp-block-wporg-chapter-list > ul' );
+	if ( container ) {
+		container.parentNode.classList.toggle( 'has-js-control' );
+
+		const parents = container.querySelectorAll( '.page_item_has_children' );
+		parents.forEach( ( item ) => {
+			// Get link, remove (will re-ad later).
+			const link = item.querySelector( ':scope > a' );
+			link.remove();
+
+			// Get submenu
+			const submenu = item.querySelector( ':scope > ul' );
+
+			// Create the toggle button.
+			const button = document.createElement( 'button' );
+			button.setAttribute( 'aria-expanded', false );
+			// button.setAttribute( 'aria-label', '' );
```

Comment:
I meant to follow up with this— this should be something like "open [page name] submenu", will update the PR tomorrow.